node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log('Hello, World!');
```

Comment:
This should console.log the 3rd element of the process.argv array

Reply:
![Screen Shot 2021-03-09 at 5 48 37 PM](https://user-images.githubusercontent.com/74885918/110563317-02175a00-8100-11eb-969c-03f366636dd3.png)